GOVERNANCE.md

```diff
@@ -0,0 +1,299 @@
+
+Our [communication channels](https://github.com/falcosecurity/community) should be used to find agreement before deciding to call a vote. Discussions can happen on any official channel. Cross-posting is recommended to give more visibility to essential topics.
+
+### Governance changes
+
+Material changes to this document must be discussed publicly on the [falcosecurity/evolution](https://github.com/falcosecurity/evolution) repository, via GitHub issues or pull requests. Any change requires a [supermajority vote](#supermajority-vote) of [Core Maintainers](#core-maintainers).
+
+The open nature of The Falco Project, its first [principle](#principles), will never be a matter of change.
+
+#### Editorial changes
+
+Editorial changes are changes that fix spelling or grammar, update links, or similar; they update the style, or keep the document up to date with obvious external changes. They do not change the intent or meaning of anything in this document. Such changes must be made via pull request, and are accepted by [lazy consensus](#consensus).
+
+### Maintainership
+
+A [Maintainer](#maintainers) may resign by notifying their willing using a GitHub issue or pull request. In such a case, they can be moved to [Emeritus Maintainers](#emeritus-maintainers) using [lazy consensus](#consensus).
+
+[Maintainers](#maintainers) can be added or moved to or from [Emeritus Maintainers](#emeritus-maintainers) with a [majority vote](#majority-vote), if the criteria in this document and the [MAINTAINERS-GUIDELINES.md](https://github.com/falcosecurity/evolution/blob/master/MAINTAINERS-GUIDELINES.md) file are met. If inactivity is the criteria, at least the past six months must be considered.
+
+In all other cases, to remove [Maintainers](#maintainers) from their role, a [supermajority vote](#supermajority-vote) is required.
+
+If the decision regards a [Core Maintainer](#core-maintainers) appointment or removal, any [Core Maintainer](#core-maintainers) can request to escalate the decision to all [Core Maintainers](#core-maintainers).
+
+In any case, the persons in question are not eligible to vote, and do not count towards the quorum.
+
+### Sensitive decisions
+
+Any sensitive matter that needs a decision to be taken privately, including but not limited to security disclosure or financial matters, may be discussed and voted on secretly if [Core Maintainers](#core-maintainers) are present and agree.
+
+In such situations, [Core Maintainers](#core-maintainers) must be notified promptly using the CNCF mailing list for Falco maintainers: [cncf-falco-maintainers@lists.cncf.io](mailto:cncf-falco-maintainers@lists.cncf.io).
+
+[Lazy consensus](#consensus) is allowed only in urgent situations that render a vote impractical.
+
+### Ordinary decisions
```

Comment:
currently, none of this governance calls for any kind of regular public meetings of maintainers or core maintainers.  Was that intentional?  Or are you having regular meetings, and just forgot to mention them here?

Reply:
We would like to delegate everything that regards community management (including meetings) to the  https://github.com/falcosecurity/community repository, as described in the first sentence of the [Community section](https://github.com/falcosecurity/evolution/blob/fb91df7167ce763e9e540d02866197eff42b097f/GOVERNANCE.md#community).

The reason for that is explained in my [comment below](https://github.com/falcosecurity/evolution/pull/169#issuecomment-1210799002).